startstoptorq.sh

```diff
@@ -0,0 +1,275 @@
+# Load the environment
+. ./setenv.sh
+
+# sets the base port for a default TorQ installation
+export KDBHDB=${PWD}/hdb/database
+export KDBWDB=${PWD}/wdbhdb
+export KDBSTACKID="-stackid ${KDBBASEPORT}"
+export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:$KDBLIB/l32
+export DEFAULTCSV=${KDBAPPCONFIG}/process.csv
```

Comment:
Can we move these into the setenv.sh script?  I think it's historical reasons that we can probably get rid of that these are in separate places.  

Also I think we should define three variables below TORQHOME.  We should have 
TORQDATA
TORQAPPHOME
 
By default, these can each just get ${PWD} as their value, but the idea is to make it easier for people to modify.  

KDBLOG can then be ${TORQDATA}/logs
KDBHDB can be ${TORQDATA}/hdb/database
KDBWDB can be ${TORQDATA}/wdbhdb
KDBAPPCODE can be ${TORQAPPHOME}/code
KDBAPPCONFIG can be ${TORQAPPHOME}/config

I think that would make it backwardly compatible, but more flexible - so a new installation just has to change TORQHOME, TORQAPPHOME and TORQDATA to have all three in different places